src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -516,3 +531,74 @@ def generate_id_verification_code() -> str:
     Generate one-time identity verification code
     """
     return str(random.choice(range(100000, 999999)))
+
+
+def _retrieve_child_results(  # pylint: disable=R0911
+    fides_connector: Tuple[str, ConnectionConfig], rule_key: str
+) -> Optional[Dict[str, Optional[List[Row]]]]:
+    """Get child access request results to add to upload."""
+    secrets = fides_connector[1].secrets
+    if not secrets:
+        logger.error(
+            "No server secrets present for privacy request %s", fides_connector[0]
+        )
+        return None
+
+    uri = secrets.get("uri")
+    if not uri:
+        logger.error("No server uri present for privacy request %s", fides_connector[0])
+        return None
+
+    username = secrets.get("username")
+    if not username:
+        logger.error(
+            "No server user name present for privacy request %s", fides_connector[0]
+        )
+        return None
+
+    password = secrets.get("password")
+    if not password:
+        logger.error(
+            "No server password present for privacy request %s", fides_connector[0]
+        )
+        return None
+
+    client = FidesClient(
+        uri=uri,
+        username=username,
+        password=password,
+    )
+    try:
+        client.login()
+    except requests.exceptions.HTTPError as e:
+        logger.error(
+            "Error logging into to child server for privacy request %s: %s",
+            fides_connector[0],
+            e,
+        )
+        return None
+
+    try:
+        request = client.authenticated_request(
+            method="get",
+            path=f"{V1_URL_PREFIX}{PRIVACY_REQUEST_TRANSFER_TO_PARENT.format(privacy_request_id=fides_connector[0], rule_key=rule_key)}",
```

Comment:
`fides_connector[0]` isn't going to be the privacy request ID, it's going to be the address corresponding to the dataset/collection for the given `FidesConnector`. i think you'll essentially need to do a look up on the full access results with this string, to grab the `PrivacyRequestResponse` dict, and look up its `"id"` field.

i realize we didn't go into this while talking earlier. i can take a closer look tomorrow morning and could probably take a stab at writing what's needed, if what i said above makes no sense :)

Reply:
I _think_ I updated the function to do what you are saying here, but if I got it wrong let me know.